kolibri/core/assets/src/utils/browserInfo.js

```diff
@@ -86,7 +86,7 @@ export const isMacWebView =
 /**
  * All web views
  */
-export const isEmbeddedWebView = isAndroidWebView || isMacWebView;
+export const isAppContext = isAndroidWebView || isMacWebView;
```

Comment:
I think we can actually just delete this now, as it shouldn't be being used anywhere (this may also mean we no longer need to define isAndroidWebView and isMacWebView).

Reply:
Alright!